hortonmachine/pom.xml

```diff
@@ -7,9 +7,8 @@
     <version>0.7.9-SNAPSHOT</version>
   </parent>
 
-	<groupId>org.jgrasstools</groupId>
 	<artifactId>jgt-hortonmachine</artifactId>
```

Comment:
did you remove the org.jgrasstools because of your custom build? 
I assume this might be a problem when building. Didn't notice that before.